ical.js

```diff
@@ -114,6 +114,26 @@ function getIanaTZFromMS(msTZName) {
   return he ? he.iana[0] : null;
 }
 
+
+let zoneTableOld = null;
+function getCurrentMSTZNameFromOldName(msTZName) {
+  if (!zoneTableOld) {
+    const p = require('path');
+    zoneTableOld = require(p.join(__dirname, 'windowsZonesOld.json'));
+  }
+  // Fixup very old data to new expectation
+  if (msTZName.startsWith('(UTC')) {
+    msTZName = '(GMT' + msTZName.substring(4);
+  }
+  if (msTZName.indexOf('&')) {
+    msTZName = msTZName.replace(/&/, 'and');
```

Comment:
Please modify to fix Code scanning comment below

Reply:
done

Reply:
Sorry, but I don't see a change here. The `replace(/&/, ...)` is still the there. See here for the reported Code Analysis issue:

https://github.com/jens-maus/node-ical/security/code-scanning/1?query=ref%3Arefs%2Fpull%2F33%2Fhead

Reply:
there is nothing wrong with that,  it is a regular expression used in replace() to change the SINGLE & to the word 'and'

Reply:
i saw that it reports that it would only change 1, but don't know how to tell the scanner that I don't need /&/g

Reply:
But wouldn't /&/g anyway the more correct approach? IMHO you want to replace all & occurances with 'and', I guess.

Reply:
I do understand. to a point.this is my contribution to your excellent package. you may accept it as is, toss it in the trash, or modify it to suit your desires. when the pr is closed, the result is your code either way.